e2e/testsuite/testsuite.go

```diff
@@ -229,23 +229,9 @@ func (s *E2ETestSuite) GetChains(chainOpts ...testconfig.ChainOptionConfiguratio
 	return path.chainA, path.chainB
 }
 
-// TODO: remove this type once the broadcast user interface aligns with ibc.Wallet
-type broadcastUser struct {
-	ibc.Wallet
-}
-
-// FormattedAddressWithPrefix implement the broadcast user interface, the FormattedAddress
-// function already includes the correct prefix.
-func (b *broadcastUser) FormattedAddressWithPrefix(prefix string) string {
-	return b.FormattedAddress()
-}
```

Comment:
this was a temporary type that is not needed anymore since the interface was updated on the interchaintest end.